resources/assets/js/videos/components/thumbnailPreview.vue

```diff
@@ -113,6 +151,15 @@ export default {
             // draw the current thumbnail to the canvas
             let context = this.thumbnailCanvas.getContext('2d');
             context.drawImage(this.sprite, sourceX, sourceY, this.thumbnailWidth, this.thumbnailHeight, 0, 0, this.thumbnailCanvas.width, this.thumbnailCanvas.height);
+        
+            // draw the hover time bar
+            context.clearRect(0, this.canvasHeight, this.canvasWidth, this.hoverTimeBarHeight);
+            context.fillStyle = this.hoverTimeStyle['bgColor'];
+            context.fillRect(0, this.canvasHeight, this.canvasWidth, this.hoverTimeBarHeight);
```

Comment:
You could also use a transparent background. Then there is one less hardcoded color.

Reply:
I'm a little confused. What should be transparent? Because in some comment above you said that the hover time background shouldn't be transparent. :sweat_smile: 

Additionally, if I remove line 157, the background becomes greyish. If I set the opacity to zero, the canvas isn't displayed any more.

Reply:
The HTML element should have the (non-transparent) background color that is configured via CSS. The canvas element can have a transparent background and only draw the time in solid color. If you remove `fillStyle` in line 157 you also have to remove `fillRect` in line 158. Otherwise it will use the `fillStyle` that is configured for the text below (in the next update) which explains what you see.